app/src/main/java/com/github/gotify/service/WebSocketConnection.java

```diff
@@ -104,7 +123,20 @@ public synchronized void close() {
         }
     }
 
+    public synchronized void scheduleReconnect(long delay, int reason) {
+        reconnectHandler.removeCallbacks(reconnectCallback);
+
+        Log.i(
+                "WebSocket: scheduling a restart in "
+                        + delay / TimeUnit.SECONDS.toMillis(1)
```

Comment:
You can use TimeUnit#convert here. `TimeUnit.SECONDS.convert(delay, TimeUnit.MILLISECONDS)`